<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeForm</class>
 <widget class="QWidget" name="RangeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Range</string>
  </property>
  <widget class="QTableWidget" name="RangeTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddPushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>181</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="NameLabel">
      <property name="text">
       <string>Имя</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="NameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Pricelabel">
      <property name="text">
       <string>Цена</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="PriceLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
